spec/requests/api/v1/rest/otus_spec.rb

```diff
@@ -6,29 +6,15 @@
 
   context 'otu/index' do
```

Comment:
Are you thinking that this `rest` pattern will isolate "true" rest endpoints from "service" endpoints?

Reply:
@mjy sorry I don't understand what you mean. Please note that OTU is not officially part of the API yet, I only refactored the spec to more or less follow the same pattern as taxon_names.

Reply:
@LocoDelAssembly Sorry for the confusion.  I didn't remember adding the `rest/` path.